localstack/services/events/provider_v2.py

```diff
@@ -79,6 +85,39 @@ def get_filtered_dict(name_prefix: str, input_dict: dict) -> dict:
     return {name: value for name, value in input_dict.items() if name.startswith(name_prefix)}
 
 
+def get_event_time(event: PutEventsRequestEntry) -> str:
+    event_time = datetime.now(timezone.utc)
+    if event_timestamp := event.get("Time"):
+        try:
+            # use time from event if provided
+            event_time = event_timestamp.replace(tzinfo=timezone.utc)
+        except ValueError:
+            # use current time if event time is invalid
+            LOG.debug(
+                "Could not parse the `Time` parameter, falling back to current time for the following Event: '%s'",
```

Comment:
this is from this fix in this pr: https://github.com/localstack/localstack/pull/8690
I cannot reproduce the bug since boto validates input and throws an error if it is not a string that can be parsed to a valid datetime object. I would keep it in for now, since it is mentioned that it has been an issue.

Reply:
Botocore validation can be disabled using `parameter_validation=False` (see [botocore docs](https://botocore.amazonaws.com/v1/documentation/api/latest/reference/config.html#botocore.config.Config)). If it's a known issue, it might be worth adding a test case in a follow-up.